bump_manifest_version.py

```diff
@@ -0,0 +1,77 @@
+import os
+
+'''
+    Usage:
```

Comment:
It would be great to also have functionality to print usage instructions if `--help` is used or not arguments are given.

Reply:
`--help` is built into into the argparse module